DOCSUP-715 network_compression_method and network_zstd_compression_level (#141)
BayoNet/ClickHouse: docs/en/operations/settings/settings.md

Default value: 0.

## network_compression_method {#network_compression_method}

> Sets the method of data compression when writing.
> Controls compression, that uses internally for communication between servers and for sending data to native clickhouse-client. ClickHouse uses the same compression methods as clickhouse-compressor does.

Reviewer: ```suggestion
Sets the method of data compression that is used for communication between servers and for sending data to native clickhouse-client.
```
